content/graph/BellmanFord.py

```diff
@@ -0,0 +1,25 @@
+"""
+Auhtor: Unknown
+Source: https://en.wikipedia.org/wiki/Bellman%E2%80%93Ford_algorithm
+Description: Computes shortest paths from $start$. Allows negative-weighted cycles
+Time: $O(VE)$
```

Comment:
```suggestion
* Auhtor: Unknown
* Source: https://en.wikipedia.org/wiki/Bellman%E2%80%93Ford_algorithm
* Description: Computes shortest paths from $start$. Allows negative-weighted cycles
* Time: $O(VE)$
```
Won't compile without this